src/lsm/manifest_level.zig

```diff
@@ -249,7 +373,7 @@ pub fn ManifestLevelType(
             visibility: Visibility,
             snapshots: []const u64,
             direction: Direction,
-            key_range: ?KeyRange,
```

Comment:
Let's preserve `key_range`. It's more explicit and clearer than only range.

Reply:
I reverted all variables names to what they were before as part of https://github.com/tigerbeetle/tigerbeetle/commit/5b58bf6ad3eeea6fbd9ba201e7369a4d32edd14d.